fem/fe/fe_h1.hpp

```diff
@@ -148,6 +149,58 @@ class H1_WedgeElement : public NodalFiniteElement
                    DenseMatrix &dshape) const override;
 };
 
+class H1_FuentesPyramidElement
```

Comment:
Can you add a reference to the paper as a comment (here or somewhere else)? I doubt people will recognize the elements by this name

Reply:
Yes, of course. I did mean to do this. Thanks for catching it.